src/ArborX_BruteForce.hpp

```diff
@@ -81,15 +81,16 @@ BruteForce<MemorySpace, BoundingVolume>::BruteForce(
   static_assert(
       KokkosExt::is_accessible_from<MemorySpace, ExecutionSpace>::value);
   Details::check_valid_access_traits(PrimitivesTag{}, primitives);
-  using Access = AccessTraits<Primitives, PrimitivesTag>;
-  static_assert(KokkosExt::is_accessible_from<typename Access::memory_space,
-                                              ExecutionSpace>::value,
+  Details::RangeAdaptor adapted_primitives(PrimitivesTag(), primitives);
+  static_assert(KokkosExt::is_accessible_from<
+                    typename decltype(adapted_primitives)::memory_space,
+                    ExecutionSpace>::value,
                 "Primitives must be accessible from the execution space");
 
   Kokkos::Profiling::pushRegion("ArborX::BruteForce::BruteForce");
 
   Details::BruteForceImpl::initializeBoundingVolumesAndReduceBoundsOfTheScene(
-      space, primitives, _bounding_volumes, _bounds);
+      space, adapted_primitives, _bounding_volumes, _bounds);
```

Comment:
I am curious to hear what other thinks.  I still think `adapt_` is better

Reply:
`adapt_primitives` sounds to me like it describes an action, i.e., a function rather than a variable.

Reply:
Longer name with no obvious benefit